lib/Serialiser/GraphViz.php

```diff
@@ -223,9 +223,9 @@ protected function escape($input)
      */
     protected function escapeAttributes($array)
     {
-        $items = '';
+        $items = array();
         foreach ($array as $k => $v) {
-            $items[] = $this->escape($k).'='.$this->escape($v);
+            array_push($items, $this->escape($k).'='.$this->escape($v));
```

Comment:
Why the change to array_push()? As the PHP docs say 

> Note: If you use array_push() to add one element to the array it's better to use $array[] = because in that way there is no overhead of calling a function.

http://us2.php.net/array_push